node-util/sbin/oo-accept-node

```diff
@@ -337,6 +337,23 @@ def check_node_public_resolution
     do_fail("#{$NODE_CONF_FILE}: PUBLIC_HOSTNAME #{pubhost} resolves to #{resolved_host}; expected #{my_ips.join '|'}") unless my_ips.member? resolved_host
 end
 
+def check_node_env
+  verbose("checking node env variables")
+
+  broker_host = $CONF.get("BROKER_HOST")
+  cloud_domain = $CONF.get("CLOUD_DOMAIN")
+
+  # check the configuration variables (BROKER_HOST & CLOUD_DOMAIN) against env variables
+  begin
+    broker_host_env = File.open("#{$CONF_DIR}/env/OPENSHIFT_BROKER_HOST", "r").read
+    cloud_domain_env = File.open("#{$CONF_DIR}/env/OPENSHIFT_CLOUD_DOMAIN", "r").read
```

Comment:
Better to use `File.read` because it's shorter and simpler and will close the FD for you.